src/js/view/view.js

```diff
@@ -856,6 +859,44 @@ function View(_api, _model) {
         return _model.get('isFloating') ? _wrapperElement : _playerElement;
     }
 
+    function _createFloatingDragUI() {
+        let dragStartX;
+        let dragStartY;
+        let playerLeft;
+        let playerTop;
+
+        const { innerHeight, innerWidth } = window;
+
+        return new UI(_wrapperElement)
+            .on('dragStart', (e) => {
+                dragStartX = e.pageX;
+                dragStartY = e.pageY;
+                playerLeft = _wrapperElement.offsetLeft;
+                playerTop = _wrapperElement.offsetTop;
+            })
+            .on('drag', (e) => {
+                if (_dragDisabled) {
+                    return;
+                }
+                let left = Math.max(playerLeft + e.pageX - dragStartX, 0);
+                let top = Math.max(playerTop + e.pageY - dragStartY, 0);
+                let right = Math.max(innerWidth - (left + _wrapperElement.clientWidth), 0);
+                let bottom = Math.max(innerHeight - (top + _wrapperElement.clientHeight), 0);
+
+                left === 0 ? right = null : left = null;
```

Comment:
If left or top is 0, unset the opposite sides.
If not, choose the opposite side and unset left and top.